setup-dev-env.sh

```diff
@@ -94,21 +120,30 @@ fi
 
 # Check installation of dev package
 if [ "$option_runtime" = "true" ]; then
-    ansible_args+=("--extra-vars" "tensorrt_install_devel=false")
-    # ROS installation type, default "desktop"
-    ansible_args+=("--extra-vars" "ros2_installation_type=ros-base")
+    ansible_args+=("--extra-vars" "ros2_installation_type=ros-base") # ROS installation type, default "desktop"
+    ansible_args+=("--extra-vars" "install_devel=false")
 else
-    ansible_args+=("--extra-vars" "tensorrt_install_devel=true")
+    ansible_args+=("--extra-vars" "install_devel=true")
 fi
 
 # Check downloading artifacts
-if [ "$option_yes" = "true" ] || [ "$option_download_artifacts" = "true" ]; then
+if [ "$target_playbook" = "autoware.dev_env.openadk" ]; then
+    if [ "$option_download_artifacts" = "true" ]; then
+        echo -e "\e[36mArtifacts will be downloaded to $option_data_dir\e[m"
+        ansible_args+=("--extra-vars" "prompt_download_artifacts=y")
+    fi
```

Comment:
When I run `./docker/build.sh`, it fails due to prompt_download_artifacts variable not being set. 
```suggestion
    if [ "$option_download_artifacts" = "true" ]; then
        echo -e "\e[36mArtifacts will be downloaded to $option_data_dir\e[m"
        ansible_args+=("--extra-vars" "prompt_download_artifacts=y")
    else
        ansible_args+=("--extra-vars" "prompt_download_artifacts=N")
    fi
```